### ISSUE TITLE
delete_files not accepting my answer

### ISSUE THREAD

**Issue by jwbensley** [jarv/cmdchallenge#69]
The command `find . -exec rm -rf "{}" \;` works on a real bash shell but not on the game (https://cmdchallenge.com/#/delete_files). However it produces the same result as the example answer `find . -delete` in that all files and subdirectories of the current directory are deleted but the current directory itself is not deleted .

```
bensley@ubuntu-laptop:/tmp/delete_files$ ls -la
total 8
drwxrwxr-x  2 bensley bensley 4096 Jan 29 21:25 .
drwxrwxrwt 18 root    root    4096 Jan 29 21:24 ..
bensley@ubuntu-laptop:/tmp/delete_files$ mkdir sub-dir
bensley@ubuntu-laptop:/tmp/delete_files$ mkdir .sub-dir
bensley@ubuntu-laptop:/tmp/delete_files$ touch file
bensley@ubuntu-laptop:/tmp/delete_files$ touch .dfile
bensley@ubuntu-laptop:/tmp/delete_files$ touch .sub-dir/.file
bensley@ubuntu-laptop:/tmp/delete_files$ find .
.
./sub-dir
./file
./.sub-dir
./.sub-dir/.file
./.dfile
bensley@ubuntu-laptop:/tmp/delete_files$ find . -exec rm -rf "{}" \;
rm: refusing to remove '.' or '..' directory: skipping '.'
find: ‘./sub-dir’: No such file or directory
find: ‘./.sub-dir’: No such file or directory
bensley@ubuntu-laptop:/tmp/delete_files$ find .
.
bensley@ubuntu-laptop:/tmp/delete_files$ ls -la
total 8
drwxrwxr-x  2 bensley bensley 4096 Jan 29 21:27 .
drwxrwxrwt 18 root    root    4096 Jan 29 21:27 ..
bensley@ubuntu-laptop:/tmp/delete_files$ 
```

The website gives the following output:

```
bash(0)> find . -exec rm -rf "{}" \;
+ find . -exec rm -rf '{}' ';'
rm: refusing to remove '.' or '..' directory: skipping '.'
find: './.config': No such file or directory
find: './var': No such file or directory
```

**Comment by jarv** [comment]
Thanks for the bug report, there is currently a limitation/issue where if any command is non-zero as this one would be it reports it as a failure regardless of the output or tests.

**Comment by jarv** [comment]
Fixed with d5b8128acd99d96164516fb68a49edb7f5905698

```
bash(0)> find . -exec rm -rf "{}" \;
rm: refusing to remove '.' or '..' directory: skipping '.'
find: './.config': No such file or directory
find: './var': No such file or directory
# 👍 👍 👍  Correct!
# You have a new challenge!
# Count the number of files in the current
# working directory. Print the number of
# files as a single integer.
# 

```
Thanks for filing the issue!
